detox/android/detox/src/full/java/com/wix/detox/espresso/web/DetoxDriverAtoms.java

```diff
@@ -0,0 +1,642 @@
+     */
+    public static Atom<Evaluation> clearElement() {
+        return new ClearElementSimpleAtom();
+    }
+
+    /**
+     * Simulates javascript key events sent to a certain element.
+     */
+    public static Atom<Evaluation> webKeys(final String text) {
+        return new WebKeysSimpleAtom(text);
+    }
+
+    /**
+     * Finds an element using the provided locatorType strategy.
+     */
+    public static Atom<ElementReference> findElement(final Locator locator, final String value) {
+        return new FindElementTransformingAtom(
+            new FindElementSimpleAtom(locator.getType(), value), castOrDie(ElementReference.class));
+    }
+
+    /**
+     * Finds the currently active element in the document.
+     */
+    public static Atom<ElementReference> selectActiveElement() {
+        return new SelectActiveElementTransformingAtom(
+            new ActiveElementSimpleAtom(), castOrDie(ElementReference.class));
+    }
+
+    /**
+     * Selects a subframe of the currently selected window by it's index.
+     */
+    public static Atom<WindowReference> selectFrameByIndex(int index) {
+        return new SelectFrameByIndexTransformingAtom(
+            new FrameByIndexSimpleAtom(index), castOrDie(WindowReference.class));
+    }
+
+    /**
+     * Selects a subframe of the given window by it's index.
+     */
+    public static Atom<WindowReference> selectFrameByIndex(int index, WindowReference root) {
+        return new SelectFrameByIndexTransformingAtom(
+            new FrameByIndexWithRootSimpleAtom(index, root),
+            castOrDie(WindowReference.class));
+    }
+
+    /**
+     * Selects a subframe of the given window by it's name or id.
+     */
+    public static Atom<WindowReference> selectFrameByIdOrName(String idOrName, WindowReference root) {
+        return new SelectFrameByIdOrNameTransformingAtom(
+            new FrameByIdOrNameWithRootSimpleAtom(idOrName, root),
+            castOrDie(WindowReference.class));
+    }
+
+    /**
+     * Selects a subframe of the current window by it's name or id.
+     */
+    public static Atom<WindowReference> selectFrameByIdOrName(String idOrName) {
+        return new SelectFrameByIdOrNameTransformingAtom(
+            new FrameByIdOrNameSimpleAtom(idOrName), castOrDie(WindowReference.class));
+    }
+
+    /**
+     * Returns the visible text beneath a given DOM element.
+     */
+    public static Atom<String> getText() {
+        return new GetTextTransformingAtom(new GetVisibleTextSimpleAtom(), castOrDie(String.class));
+    }
+
+    /**
+     * Returns {@code true} if the desired element is in view after scrolling.
+     */
+    public static Atom<Boolean> webScrollIntoView() {
+        return new WebScrollIntoViewAtom(new WebScrollIntoViewSimpleAtom(), castOrDie(Boolean.class));
+    }
+
+    /**
+     * Finds multiple elements given a locator strategy.
+     */
+    public static Atom<List<ElementReference>> findMultipleElementsDetox(
```

Comment:
except for this method I see lots of methods here that we're not using, copied from the original that we're still using. Can we keep only the specific custom method we're using?

Reply:
This is still WIP @asafkorem